core/src/auctions.ts

```diff
@@ -173,12 +195,12 @@ export const enrichAuction = async function (
     // enrich them with price drop
     const auctionWithPriceDrop = await enrichAuctionWithPriceDrop(auctionWithStatus);
 
-    // enrich them with market values
-    const auctionWithMarketValue = await enrichAuctionWithMarketValues(auctionWithPriceDrop, network);
+    // enrich them with market data
+    const auctionWithMarketData = await enrichAuctionWithMarketData(auctionWithPriceDrop, network);
 
     // enrich with profit and fee calculation
     const fees = await getApproximateTransactionFees(network);
-    const auctionWithFees = await enrichAuctionWithTransactionFees(auctionWithMarketValue, fees, network);
+    const auctionWithFees = await enrichAuctionWithTransactionFees(auctionWithMarketData, fees, network);
```

Comment:
How gross profit can be calculated inside `enrichAuctionWithMarketData` if `auctionWithFees` is only calculated here? I imagine that the order needs to be changed?

Reply:
you're right

Reply:
But the same applies to `netProfit`: we suppose to calculate it for every market, right?

Reply:
This wasn't addressed?